src/modules/playback-engine/playback-engine.ts

```diff
@@ -662,24 +635,22 @@ class Engine implements IPlaybackEngine {
     const { duration, currentTime } = this._output;
     let data;
 
-    if (this._adapterStrategy.attachedAdapter) {
-      data = this._adapterStrategy.attachedAdapter.debugInfo;
+    if (this._output.attachedAdapter) {
```

Comment:
Let's make methods `getDebugInfo` in `output` and move this logic inside it

Reply:
done